array.c

```diff
@@ -4373,65 +4373,102 @@ take_items(VALUE obj, long n)
 
 /*
  *  call-seq:
- *    array.zip(*other_arrays) -> new_array
- *    array.zip(*other_arrays) {|other_array| ... } -> nil
+ *    zip(*objects) -> new_array
+ *    zip(*objects) {|other_array| ... } -> nil
  *
- *  When no block given, returns a new +Array+ +new_array+ of size <tt>self.size</tt>
- *  whose elements are Arrays.
+ *  Operates on a collection of +operands+, each of which is an array
+ *  based on one of the given +objects+.
```

Comment:
I'm confused by this description. It's not clear what `operands` is referring to.